crates/next-core/src/manifest.rs

```diff
@@ -0,0 +1,103 @@
+use anyhow::Result;
+use indexmap::IndexSet;
+use mime::APPLICATION_JSON;
+use turbo_tasks::primitives::StringsVc;
+use turbo_tasks_fs::File;
+use turbopack_core::asset::AssetContentVc;
+use turbopack_dev_server::source::{
+    combined::CombinedContentSourceVc, conditional::ConditionalContentSourceVc, ContentSource,
+    ContentSourceContent, ContentSourceData, ContentSourceResultVc, ContentSourceVc,
+};
+use turbopack_node::{
+    node_api_source::NodeApiContentSourceVc, node_rendered_source::NodeRenderContentSourceVc,
+};
+
+#[turbo_tasks::value(shared)]
+pub struct DevManifestContentSource {
```

Comment:
Please add some documentation to this struct and its public items.

Reply:
done